app/scripts/utils/fileExtention.js

```diff
@@ -0,0 +1,14 @@
+export default function fileExtention (name, defaultExt = '?') {
```

Comment:
should probably be fileExtension, also lets use file-extension.js for the filename, some of us have cas insensitive filesystems, right @whyrusleeping ;)